[Release] Prepare Changelog for 8.8.1 (#35695)
elastic/beats: CHANGELOG.asciidoc

:issue: https://github.com/elastic/beats/issues/
:pull: https://github.com/elastic/beats/pull/

[[release-notes-8.8.1]]
=== Beats version 8.8.1
https://github.com/elastic/beats/compare/v8.8.0\...v8.8.1[View commits]

==== Bugfixes

*Affecting all Beats*

- 'add_cloud_metadata' processor - add cloud.region field for GCE cloud provider
- 'add_cloud_metadata' processor - update azure metadata api version to get missing `cloud.account.id` field

*Filebeat*

> - Fixes "Can only start an input when all related states are finished" error when running under Elastic-Agent {pull}35250[35250] {issue}33653[33653]
> - [system] sync system/auth dataset with system integration 1.29.0. {pull}35581[35581]
> - Fix filestream false positive log error "filestream input with ID 'xyz' already exists" {issue}31767[31767]
> - Fix error when trying to use `include_message` parser {issue}35440[35440]

Reviewer: ```suggestion
- Fix "Can only start an input when all related states are finished" error when running under Elastic Agent {pull}35250[35250] {issue}33653[33653]
- [system] Sync system/auth dataset with system integration 1.29.0. {pull}35581[35581]
- Fix filestream false positive log error "filestream input with ID 'xyz' already exists" {issue}31767[31767]
- Fix error when trying to use `include_message` parser {issue}35440[35440]
```

- Add `event.category` and `event.type` to Sysmon module for EventIDs 8, 9, 19, 20, 27, 28, 255. {pull}35193[35193]

[[release-notes-8.7.1]]
=== Beats version 8.7.1
https://github.com/elastic/beats/compare/v8.7.0\...v8.7.1[View commits]

==== Bugfixes

*Affecting all Beats*

> - Fix Beats started by agent not respecting the allow_older_versions: true configuration flag {issue}34227[34227] {pull}34964[34964]

Reviewer 2: This has been released already, see it on Mar 30 towards the bottom of https://github.com/elastic/beats/commits/v8.7.1

I think the one below has also been released already.

I can see this in the 8.7.1 changelog that was forward ported to main which is odd: https://github.com/elastic/beats/pull/35208/files

Maybe it was merged to main after 8.8 was created, in which case we need to backport that PR to main and then regenerate this.

Reviewer 3: What you are mentioning is in the 8.7.1 section, as everything was missing in this branch, I added the content of 8.7.0 and 8.7.1 release here and deleted duplicates in the 8.8.1 section.

Reviewer 2: Ah I was expecting to only see 8.8 relate content, thanks.

> - Fix Beats started by agent not respecting the allow_older_versions: true configuration flag {issue}34227[34227] {pull}34964[34964]
> - Beats started by agent now configure one global instance of each default add_x_metadata processor instead of creating one processor per input. Fixes a significant performance degradation for the aws-s3 input. {issue}35000[35000] {pull}35031[35031]

Reviewer: ```suggestion
- Fix Beats started by Elastic Agent not respecting the `allow_older_versions: true` configuration flag {issue}34227[34227] {pull}34964[34964]
- Beats started by Elastic Agent now configures one global instance of each default `add_x_metadata` processor instead of creating one processor per input. Fixes a significant performance degradation for the aws-s3 input. {issue}35000[35000] {pull}35031[35031]
```

- Fix accidental error overwrite in defer statement in entityanalytics Azure AD input. {issue}35153[35153] {pull}35169[35169]

*Heartbeat*

- Fix issue using projects in airgapped environments by disabling npm audit. {pull}34936[34936]

*Packetbeat*

- Fix documentation for `flows.period` related to flow reporting. {pull}35009[35009]

==== Added

*Affecting all Beats*

- Allow users to enable features via configuration, starting with the FQDN reporting feature. {issue}1070[1070] {pull}34456[34456]

*Winlogbeat*

- Add note in documentation about 21 event ID clause limit {issue}35048[35048] {pull}35049[35049]

[[release-notes-8.7.0]]
=== Beats version 8.7.0
https://github.com/elastic/beats/compare/v8.6.2\...v8.7.0[View commits]

==== Bugfixes

*Affecting all Beats*

- Fix dropped events when monitor a beat under the agent and send its `Host info` log entry. {pull}34599[34599]
- Fix panics when a processor is closed twice {pull}34647[34647]
- Update elastic-agent-system-metrics to v0.4.6 to allow builds on mips platforms. {pull}34674[34674]

*Filebeat*

> - [Auditbeat System Package] Added support for Apple Silicon chips. {pull}34433[34433]

Reviewer: ```suggestion

- [Auditbeat System Package] Added support for Apple Silicon chips. {pull}34433[34433]
```
We need a blank line here to make the items render as a list.
